### PR TITLE
Update shimmer-firefly-governance-guide.md

### PR DESCRIPTION
**Pull request by Phyloiota**
Grammar, spelling and Make the guide generic for IOTA and Shimmer

# Description of change

Made the guide generic to both Firefly versions

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/learn/governance/shimmer-firefly-governance-guide.md
Comment (Dr-Electron) [inline docs/learn/governance/shimmer-firefly-governance-guide.md:1]: You forgot to rename this file

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: docs/learn/governance/shimmer-firefly-governance-guide.md
Comment (Phyloiota) [reply docs/learn/governance/shimmer-firefly-governance-guide.md:1]: updated, thanks

**Review by Dr-Electron — APPROVED** [review]

**Review by Dr-Electron — APPROVED** [review]
I fixed some broken links and now it looks good to me.
Keep in mind that now some links on the wiki will be changed with all those changes.

Also you might want to change the image (shimmer-gov.png)

**Comment by Phyloiota** [comment]
updated the images as suggested

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**